notebook/templates/edit.html

```diff
@@ -95,5 +109,12 @@
 
     {{super()}}
 
+<script>
+$(document).ready(function(){
+    $("#toggle_header").click(function(){
```

Comment:
Thank you for your experience. Just wasn't sure where you guys wanted the javascript. Yea I remember putting in stuff I didn't need thought I got rid of it. 

Also npm run build:watch does not work for me. I'm just using sublime save and refresh page. Probably a tad slower but that's alright for now.

I'll put the changes soon.